### ISSUE TITLE
Support Open Graph

### ISSUE THREAD

**Issue by penibelst** [gjtorikian/html-proofer#111]
The [Open Graph](http://ogp.me/) protocol requires two properties for every page we could check:

* `og:image` - An image URL which should represent your object within the graph.
* `og:url` - The canonical URL of your object that will be used as its permanent ID in the graph, e.g., "http://www.imdb.com/title/tt0117500/".

Example:

```html
<meta property="og:url" content="http://www.example.com/" />
<meta property="og:image" content="http://www.example.com/image.png" />
```

**Comment by peternewman** [comment]
So will this be in 3.2.1 or 3.3.0 or something? Is there a rough ETA for a release date so we can start using it?

**Comment by gjtorikian** [comment]
Sure, I just released it as 3.3.0.
